packages/console/src/onboarding/pages/CreateTenant/index.tsx

```diff
@@ -0,0 +1,107 @@
+import { joinPath } from '@silverhand/essentials';
+import { useContext } from 'react';
+import { Controller, FormProvider, useForm } from 'react-hook-form';
+
+import { useCloudApi } from '@/cloud/hooks/use-cloud-api';
+import ActionBar from '@/components/ActionBar';
+import { type CreateTenantData } from '@/components/CreateTenantModal/types';
+import Region, { RegionName } from '@/components/Region';
+import { isDevFeaturesEnabled } from '@/consts/env';
+import { TenantsContext } from '@/contexts/TenantsProvider';
+import Button from '@/ds-components/Button';
+import DangerousRaw from '@/ds-components/DangerousRaw';
+import FormField from '@/ds-components/FormField';
+import OverlayScrollbar from '@/ds-components/OverlayScrollbar';
+import RadioGroup, { Radio } from '@/ds-components/RadioGroup';
+import TextInput from '@/ds-components/TextInput';
+import useTenantPathname from '@/hooks/use-tenant-pathname';
+import * as pageLayout from '@/onboarding/scss/layout.module.scss';
+import { OnboardingPage, OnboardingRoute } from '@/onboarding/types';
+import { trySubmitSafe } from '@/utils/form';
+
+type CreateTenantForm = Omit<CreateTenantData, 'tag'>;
```

Comment:
Why should we ignore `tag` in this case?

Reply:
because currently it's always dev